docs/examples/201/aci-sftp-files/main.bicep

```diff
@@ -0,0 +1,130 @@
+  name: identityName
+  location: location
+}
+
+resource roleAssignment 'Microsoft.Authorization/roleAssignments@2020-04-01-preview' = {
+  name: roleDefinitionName
+  properties: {
+    roleDefinitionId: roleDefinitionId
+    principalId: managedIdentity.properties.principalId
+    principalType: 'ServicePrincipal'
+  }
+}
+
+resource storageAccount 'Microsoft.Storage/storageAccounts@2020-08-01-preview' = {
+  name: storageAccountName
+  location: location
+  sku: {
+    name: storageAccountType
+  }
+  kind: 'StorageV2'
+  dependsOn: [
+    roleAssignment // need to create a slight delay for the roleAssignment to replicate before the deployment script can run
+  ]
+}
+
+resource deploymentScript 'Microsoft.Resources/deploymentScripts@2020-10-01' = {
+  name: scriptName
+  location: location
+  kind: 'AzurePowerShell'
+  identity: {
+    type: 'UserAssigned'
+    userAssignedIdentities: {
+      '${managedIdentity.id}': {}
+    }
+  }
+  properties: {
+    forceUpdateTag: '1'
+    azPowerShellVersion: '3.0'
+    arguments: ' -storageAccountName ${storageAccount.name} -fileShareName ${fileShareName} -resourceGroupName ${resourceGroup().name}'
+    scriptContent: 'param([string] $storageAccountName, [string] $fileShareName, [string] $resourceGroupName) Get-AzStorageAccount -StorageAccountName $storageAccountName -ResourceGroupName $resourceGroupName | New-AzStorageShare -Name $fileShareName'
+    timeout: 'PT5M'
+    cleanupPreference: 'OnSuccess'
+    retentionInterval: 'P1D'
+  }
+}
+
+resource containerGroup 'Microsoft.ContainerInstance/containerGroups@2019-12-01' = {
+  name: sftpContainerGroupName
+  location: location
+  dependsOn: [
+    deploymentScript // Need to create the fileShare before creating the container.
+  ]
+  properties: {
+    containers: [
+      {
+        name: sftpContainerName
+        properties: {
+          image: sftpContainerImage
+          environmentVariables: [
+            {
+              name: 'SFTP_USERS'
+              value: sftpEnvVariable
+            }
+          ]
+          resources: {
+            requests: {
+              cpu: 2
+              memoryInGB: 1
+            }
+          }
+          ports: [
+            {
+              port: 22
+            }
+          ]
+          volumeMounts: [
+            {
+              name: 'sftpvolume'
+              mountPath: '/home/${sftpUser}/upload'
+              readOnly: false
+            }
+          ]
+        }
+      }
+    ]
+    osType: 'Linux'
+    ipAddress: {
+      type: 'Public'
+      ports: [
+        {
+          protocol: 'TCP'
+          port: 22
+        }
+      ]
+    }
+    restartPolicy: 'OnFailure'
+    volumes: [
+      {
+        name: 'sftpvolume'
+        azureFile: {
+          readOnly: false
+          shareName: fileShareName
+          storageAccountName: storageAccount.name
+          storageAccountKey: listKeys(storageAccountName, '2020-08-01-preview').keys[0].value
```

Comment:
you should get the name and apiVersion from the symbolic name:

```
storageAccountKey: listKeys(storageAccount.name, storageAccount.apiVersion).keys[0].value
```

Reply:
Done